admin_manual/configuration/files/external_storage/google.rst

```diff
@@ -14,97 +14,135 @@ If you already have a Google account, such as Groups, Drive, or Mail, you can
 use your existing login to log into the Google Cloud Console. After logging in 
 click  the **Create Project** button.
 
-.. figure:: images/google-drive.png
-   :alt: Google Drive app "create project" screen
+.. figure:: images/google_drive/001.png
+   :alt: Google Cloud Console
+
+.. figure:: images/google_drive/002
+   :alt: Create Project
+
 
 Give your project a name, and either accept the default **Project ID** or 
-create 
-your own, then click the **Create** button.
+create your own, then click the **Create** button. For this example a random name was choosen "owncloud-04-27". Feel free to choose your own name.
 
-.. figure:: images/google-drive1.png
-   :alt: New project creation screen.
+.. figure:: images/google_drive/003
+   :alt: Choose a name
+ 
+After your project is created, click on the notifications bell and select your project.
 
-You'll be returned to your dashboard.
+.. figure:: images/google_drive/004
+   :alt: Notification bell
 
-.. figure:: images/google-drive2.png
-   :alt: Project dashboard, with all project options.
-   :scale: 75% 
-   
-Google helpfully highlights your next step in blue, the **Use Google APIs** 
-box. Make sure that your new project is selected, click on **Use Google APIs** , 
-and it takes you to Google's APIs screen. There are many Google APIs; look for 
-the **Google Apps APIs** and click **Drive API.**
+Go to Api overview to select google's API.
 
-.. figure:: images/google-drive3.png
-   :alt: Drive API link.
+.. figure:: images/google_drive/005
+   :alt: API
 
-**Drive API** takes you to the API Manager overview. Click the blue **Enable 
-API** button.
+.. figure:: images/google_drive/006
+   :alt: Google API
 
-.. figure:: images/google-drive4.png
-   :alt: API dashboard.
+.. figure:: images/google_drive/007
+   :alt: Enable
 
-Now you must create your credentials, so click on **Go to credentials**.
-   
-.. figure:: images/google-drive5.png
-   :alt: Create Client ID screen.
+Now you must create your credentials.
    
-For some reason Google warns us again that we need to create credentials. We 
-will use OAuth 2.0.
+.. figure:: images/google_drive/008
+   :alt: Create Credentials
 
-.. figure:: images/google-drive6.png
-   :alt: Another warning to create credentials.
-   
-Now we have to create a consent screen. This is the information in the screen 
-Google shows you when you connect your new Google app to ownCloud the first 
-time. Click **Configure consent screen**. Then fill in the required form fields. 
-Your logo must be hosted, as you cannot upload it, so enter its URL. When 
-you're finished click **Save**.
+First select Web Browser and User data.
```

Comment:
s/First select Web Browser and User data./First, select "Web Browser" and "User data"./g